apps/patient/src/views/Pharmacy.tsx

```diff
@@ -548,14 +548,16 @@ export const Pharmacy = () => {
     trackSelectedPharmacyRank(selectedId, allPharmacies);
 
     try {
+      const patientSelectedPrice = !!allPharmacies.find((p) => p.id === selectedId)?.price;
```

Comment:
Is `allPharmacies` only a list of pharmacies with prices? or could that also be pharmacies from the distance list?

Reply:
reverted to the way we're doing it for DD right now, `sortBy === 'price'`